cmsController.php

```diff
@@ -8,7 +8,7 @@
  * @param $db to select from
  * @return assoc array of about sections and content
  */
-function FillAbout ($db): {
+function FillAbout (PDO $db): array{
```

Comment:
is there a space after the colon? I do `(PDO $db):array`

Reply:
resolved